api/src/App/Routes.php

```diff
@@ -35,6 +35,7 @@ function () use ($app, $authMiddleware) {
         function () use ($app, $authMiddleware) {
             $app->get('[/]', 'App\Controller\Department\ListDepartments');
             $app->post('[/]', 'App\Controller\Department\PostDepartment');
+            $app->put('/{id}', 'App\Controller\Department\PutDepartment');
```

Comment:
I personally like the use if `{id}` but in quite a few other places we are using `{name}`.  I think the APIs will also accept a name or ID,  do we want to try to be consistant here or is that overkill?

Reply:
We opt for `{name}` here if we're looking for consistency. I'll check to see if anything actually passes in `{name}` to the other endpoints for department and make the switch if needed. Ideally we do ID based lookup only because name could be ambiguous.

Reply:
I took a look through the other endpoints that are using `{name}` syntax for Announcements and Deadlines. I poked around in the UI a bit and I confirmed through debugging that they are using the Department ID value. I know all of the ConCom List stuff should be using the Department ID value as well.

Is it worth considering the move to prefer ID for Department here and we can eventually update the other endpoints to ID numbers only?

Reply:
Lets make a low priority task to try to standardize our language and make {id} for all places we are using IDs.